src/poliastro/twobody/orbit.py

```diff
@@ -216,12 +216,29 @@ def h_vec(self):
         h_vec = np.cross(self.r.to(u.km).value, self.v.to(u.km / u.s)) * u.km ** 2 / u.s
         return h_vec
 
+    @property
+    def h_mom(self):
+        """Angular momentum. """
+        h_mom = np.linalg.norm(self.h_vec) * u.km ** 2
+        return h_mom
+
     @property
     def arglat(self):
         """Argument of latitude. """
         arglat = (self.argp + self.nu) % (360 * u.deg)
         return arglat
 
+    @property
+    def M(self):
+        """Mean anomaly. """
+        return nu_to_M(self.nu, self.ecc)
+
+    @property
+    def t_p(self):
+        """Elapsed time since latest perifocal passage. """
+        t_p = self.period * self.M / (360 * u.deg)
```

Comment:
Interesting! I expected some subtraction from `self.epoch` but I guess this also works.

Also, perhaps instead of `/ (360 * u.deg)` we can have `/ (1 * u.cycle)`?

Reply:
I started of like you said by subtracting. But I then saw that Curtis (as well as some various sources) expressed it as elapsed time rather than epoch which makes sense (and the result can then easily be subtracted).

Strangely, ``1 * u.cycle`` doesn't yield correct results